torchao/float8/float8_ops.py

```diff
@@ -481,3 +483,36 @@ def copy_fp8(aten_op, args, kwargs=None):
         )
     else:
         raise RuntimeError("Unsupported semantics for copy_ in Float8Tensor")
+
+
+def mm_float8_emulated(
+    m1,  # input 1 data
+    s1,  # input 1 scale
+    m2,  # input 2 data
+    s2,  # input 2 scale
+    dtype3,  # output dtype
+):
+    # naive implementation: dq -> op -> q
+    m1_fp32 = m1.float() / s1
+    m2_fp32 = m2.float() / s2
+    m3_fp32 = torch.mm(m1_fp32, m2_fp32)
+
+    return m3_fp32.to(dtype3)
+
+
+# ATen op placeholders
```

Comment:
can delete from here to the bottom of the file after the top function is inlined